admin_manual/configuration_files/files_locking_enabling.rst

```diff
@@ -16,4 +17,28 @@ underlying filesystem.
 The Files Locking app has no configuration options; all you need to do is 
 enable or disable it on your Apps page.
 
-.. figure:: ../images/files_locking_app.png
\ No newline at end of file
+.. figure:: ../images/files_locking_app.png
+
+We recommend using `Redis <http://redis.io/>`_ as your ownCloud memcache when 
+you enable the Files Locking app. Memcached, the popular distributed memory 
+caching system, is not suitable for a file locking app because it is designed 
+to improve dynamic Web site performance. It is not designed to store locks, and 
+data can disappear from the cache at any time. Redis is more than an object 
+cache like Memcached; it is also a key-value store, so it guarantees that 
+cached objects are available for as long as they are needed. Redis is available 
+on most Linux distributions, and requires a simple configuration in your 
+``config.php`` file, like this example::
+
+ 'memcache.local' => '\OC\Memcache\redis',
```

Comment:
it doesn't matter as long as the character following the `\` doesn't have a special escaped meaning